package/gluon-core/files/lib/gluon/domain_changed.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+domain=$(uci get gluon.@system[0].domain)
```

Comment:
The variable is not populated by default, also `gluon` would be an entirely new section, while `gluon-core` already exists.

Reply:
Yes, neoraider said, that he want to merge all the `gluon-*` sections in one `gluon` section in near future. And I already use this new section. But I'm currently not sure how to introduce sections. I'll add a todo for this. 

But you pointed me to an other error. It should be `uci get gluon.system.domain`. Thanks.